src/atomate2/vasp/flows/mp.py

```diff
@@ -0,0 +1,65 @@
+"""
+Module defining flows for Materials Project r2SCAN workflows.
+
+Reference: https://doi.org/10.1103/PhysRevMaterials.6.013801
+"""
+
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from typing import TYPE_CHECKING
+
+from jobflow import Flow, Maker
+
+if TYPE_CHECKING:
+    from pathlib import Path
+
+    from pymatgen.core.structure import Structure
+
+    from atomate2.vasp.jobs.base import BaseVaspMaker
+
+from atomate2.vasp.jobs.mp import MPPreRelaxMaker, MPRelaxMaker
+
+__all__ = ["MP2023RelaxMaker"]
+
+
+@dataclass
+class MP2023RelaxMaker(Maker):
+    """
+    Maker to perform a VASP r2SCAN relaxation workflow with MP settings.
+
+    Parameters
+    ----------
+    name : str
+        Name of the flows produced by this maker.
+    pre_relax_maker : .BaseVaspMaker
+        Maker to generate the first relaxation.
+    relax_maker : .BaseVaspMaker
+        Maker to generate the second relaxation.
+    """
+
+    name: str = "MP 2023 Relax"
+    pre_relax_maker: BaseVaspMaker = field(default_factory=MPPreRelaxMaker)
```

Comment:
Should the name be more descriptive e.g. "MP metaGGA Relax" or "MP r2SCAN Relax"?

I'm also not sure how I feel about the 2023 in the name. Since the papers came out in 2022, is there possibility for that to cause confusion?

Reply:
See comment below about not tying the name to a given setting, but I agree that 2023 is confusing. We can switch to 2022, although I don't know that the year is ideal. This was mostly inspired by the compatibility scheme naming. 

Any alternate suggestions given this info?

Reply:
Actually, I like `MPMetaGGARelax` for the class name. I'll use that for now.